openstudiocore/src/utilities/core/Finder.hpp

```diff
@@ -103,7 +103,7 @@ namespace openstudio{
     typename std::vector<T>::const_iterator it;
     it = find_if(vec.begin(),vec.end(),finder);
     if (it != vec.end()) {
-      index = it - vec.begin();
+      index = (int)(it - vec.begin());

```

Comment:
Yes, to answer the general question, 64bit MSVC is the only platform where `sizeof(int) != sizeof(size_t)`

The proper thing to do is to actually correct the errors, because there's probably a lurking overflow somewhere. This can generally be done with using `auto` more (which this file could benefit from immensely it appears to be at least 25% longer then it needs to be).

Practically, the overflow issues will never be a problem for you. Technically, you are arbitrarily limiting the API to working with 2 billion ish objects instead of 2^64 objects.